### PR TITLE
Add Austria

### PR DESCRIPTION
**Pull request by gregn610**
Add Austria

### CODE REVIEW

File: workalendar/europe/austria.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from workalendar.core import WesternCalendar, ChristianMixin
+
+
+class Austria(WesternCalendar, ChristianMixin):
+    "Austria"
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5,   1, "National Holiday"),  # Staatsfeiertag
```
Comment (brunobord) [inline workalendar/europe/austria.py]: isn't this one "Labour Day"?

**Comment by brunobord** [comment]
there's a tiny conflict in the CHANGELOG, because I've merged the #155 PR

File: workalendar/europe/austria.py (hunk 1 above)
Comment (gregn610) [inline workalendar/europe/austria.py]: Dunno, could be, I'm not any kind of expert on these things :)
I generally go with with Wikipedia on these things.
[Austria.info](http://www.austria.info/uk/service-facts/practical-information/public-holidays-daylight-savings-time) looks kinda official and the [literal translation of Staatsfeiertag](https://translate.google.com/#de/en/Staatsfeiertag) is National Day.

Comment (brunobord) [inline workalendar/europe/austria.py]: right, we'll keep it that way until someone finds a better wording ; what's important is the date, the label can wait.

**Comment by brunobord** [comment]
so you still have to solve the conflicts with CHANGELOG

**Comment by brunobord** [comment]
great one! althought it would have been better to have solved the conflict using "rebase" rather than "merge" to keep your branch clean ; if you need some help to do so for your future contributions, I'd be glad to.

**Comment by gregn610** [comment]
Yeah, sorry about that, I'm still figuring out the git workflows.

**Pull request merged**